presto-tpch/pom.xml

```diff
@@ -79,5 +79,12 @@
             <artifactId>testng</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
```

Comment:
Do we need to bring in the new dependency--can we just use the HyperLogLog implementation already used in Presto?